src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -57,6 +57,7 @@ export type Layers = {
   readonly overriddenInfobox?: OverriddenInfobox;
   readonly overriddenProperties?: { [id: string]: any };
   /** Selects the layer with the specified ID; if the ID is undefined, the currently selected later will be deselected. */
+  readonly getLayersInViewport: () => void;
```

Comment:
```suggestion
  readonly layersInViewport: () => Layer[];
```

And also following method should be also added:

```ts
type Rect = {
  north: number;
  south: number;
  east: number;
  west: number;
}

type Camera = {
  // ...
  readonly viewport: () => Rect | undefined;
}
```